code/game/objects/structures/watercloset.dm

```diff
@@ -389,7 +389,8 @@
 			return 1
 		busy = FALSE
 		O.wash(CLEAN_WASH)
-		O.acid_level = 0
+		create_reagents(5)
+		reagents.add_reagent(dispensedreagent, 5)
```

Comment:
Why is it creating a new reagent datum every wash?

Reply:
I don't know. I don't know why that change is there either. I'll look into it.